utils/data-migration/bss-ewcs/migrate-users.js

```diff
@@ -17,7 +18,7 @@ const loadUsersFromFile = () => {
 
 const generateRoles = (v1Role) => {
   if (v1Role === 'Maker/Checker') {
```

Comment:
Are we keeping this in for legacies sake?

Reply:
Yep exactly, I don't think we can remove this check because the data being migrated from will still have this role